.travis.yml

```diff
@@ -1,34 +1,19 @@
 ---
 language: node_js
 node_js:
-  - "0.12"
+  - "8"
```

Comment:
@ef4 does this mean that older Node versions are no longer supported? if so this should have probably been released as a breaking change 🤔

Reply:
CI did not run at all. Also package.json states:
```
  "engines": {
    "node": ">= 4"
  },
```
 I guess it's only an improvement then.

Reply:
shouldn't we run CI with Node v4 then?

Reply:
I could add a Node@4 job if needed.

Reply:
😂 you answered 9 seconds before me